src/colophon.adoc

```diff
@@ -13,43 +14,94 @@ The document contains the following versions of the RISC-V ISA modules:
 [%autowidth,float="center",align="center",cols="^,<,^",options="header"]
 |===
 |Base |Version |Status
-|*RVWMO* |*2.0* |*Ratified*
 |*RV32I* |*2.1* |*Ratified*
-|*RV64I* |*2.1* |*Ratified*
 |*RV32E* |*2.0* |*Ratified*
 |*RV64E* |*2.0* |*Ratified*
+|*RV64I* |*2.1* |*Ratified*
 |_RV128I_ |_1.7_ |_Draft_
 
 h|Extension h|Version h|Status
 
+|*Zifencei* |*2.0* |*Ratified*
+|*Zicsr* |*2.0* |*Ratified*
+|*Zicntr* |*2.0* |*Ratified*
+|*Zihintntl* |*1.0* |*Ratified*
+|*Zihintpause* |*2.0* |*Ratified*
+|_Zimop_ | _0.1_ | _Draft_
+|*Zicond* | *1.0* |*Ratified*
 |*M* |*2.0* |*Ratified*
+|*Zmmul* |*1.0* |*Ratified*
 |*A* |*2.1* |*Ratified*
+|*Zawrs* |*1.01* |*Ratified*
+|*Zacas* |*1.0* |*Ratifed*
+|*RVWMO* |*2.0* |*Ratified*
+|*Ztso* |*1.0* |*Ratified*
+|*CMO* |*1.0* |*Ratified*
 |*F* |*2.2* |*Ratified*
 |*D* |*2.2* |*Ratified*
 |*Q* |*2.2* |*Ratified*
-|*C* |*2.0* |*Ratified*
-|_Counters_ |*2.0* |*Ratified*
-|_P_ |_0.2_ |_Draft_
-|*V* |*1.0* |*Ratified*
-|*Zicsr* |*2.0* |*Ratified*
-|*Zifencei* |*2.0* |*Ratified*
-|*Zihintpause* |*2.0* |*Ratified*
-|*Zihintntl* |*1.0* |*Ratified*
-|*Zfa* |*1.0* |*Ratified*
 |*Zfh* |*1.0* |*Ratified*
 |*Zfhmin* |*1.0* |*Ratified*
+|*Zfa* |*1.0* |*Ratified*
 |*Zfinx* |*1.0* |*Ratified*
 |*Zdinx* |*1.0* |*Ratified*
 |*Zhinx* |*1.0* |*Ratified*
 |*Zhinxmin* |*1.0* |*Ratified*
-|*Zmmul* |*1.0* |*Ratified*
-|*Ztso* |*1.0* |*Ratified*
+|*C* |*2.0* |*Ratified*
+|*Zc* |*1.0* |*Ratified*
```

Comment:
Zc isn't an actual extension; renaming this one to Zce would suffice.

Reply:
Done.